### ISSUE TITLE
calendar.events.list has wrong scope

### ISSUE THREAD

**Issue by bryanburgers** [Byron/google-apis-rs#235]
Running `calendar.events.list` fails when it isn't given a scope.

```rust
let result = hub
              .events()
              .list(&calendar.id)
              .time_min(&start.to_rfc3339())
              .time_max(&end.to_rfc3339())
              .max_results(100)
              .single_events(true)
              .order_by("startTime")
              .doit();
```

fails with

```rust
Err(MissingToken(StringError { error: "Invalid Scope: \'no description provided\'\n" }))
```

However, by adding `.add_scope(gcal::Scope::Readonly)`, the call succeeds:

```rust
let result = hub
              .events()
              .list(&calendar.id)
              .add_scope(gcal::Scope::Readonly)
              .time_min(&start.to_rfc3339())
              .time_max(&end.to_rfc3339())
              .max_results(100)
              .single_events(true)
              .order_by("startTime")
              .doit();
```

It appears that the default scope if none is given is `gcal::Scope::EventReadonly` (https://docs.rs/google-calendar3/1.0.10+20190702/src/google_calendar3/lib.rs.html#11260). I wasn't able to figure out how to change the default in `/etc/api/calendar/v3/calendar-api.json` (or even if the default should be changed).

---

Note: My application only uses `calendar.calendarList.list` and `calendar.events.list`; the former works without specifying a scope, the latter does not.

I once had an application running with 1.0.7+something and it worked without adding a scope. I believe 1.0.8+something did not work without adding a scope. I just updated to 1.0.10 and found the scope workaround.

**Comment by Byron** [comment]
Thanks for the detailed post and the code to reproduce the issue, it's much appreciated!

What you have discovered is a well-meant feature that turns out to be a surprise sometimes, like here. The code generator guesses default scopes, and sometimes is just wrong, leading to hard-to-understand error messages.

The workaround you found is the one to use in this case.

[We have been discussing a better way](https://gitter.im/google-apis-rs/community?at=5d646198c8228962accd5f4e) to do this, which will be implemented in the next version of [google-apis](https://github.com/google-apis-rs/generator).

**Comment by Byron** [comment]
Maybe you could try with the latest version of the crates now available with async support in v2.0 or the next generation of [API generators](https://github.com/google-apis-rs).

As this repository is now in _maintenance mode_, this issue is probably not going to be fixed unless it is contributed.
